cdap-ui/app/features/admin/routes.js

```diff
@@ -227,8 +227,23 @@ angular.module(PKG.name + '.feature.admin')
                 parent: 'admin.namespace.detail',
                 url: '/:appId',
                 templateUrl: '/assets/features/admin/templates/namespace/app-metadata.html',
-                controller: 'AdminNamespaceAppMetadataController'
-              });
+                controller: 'AdminNamespaceAppMetadataController',
+                resolve: {
+                    source: function () {
```

Comment:
If we add a parameter to resolve object in a route we usually add a 'r' as a prefix (like rDashboardModel or rNsList) may be we should follow that convention? Just noticing it. We need to probably change in adapters too.